tests/wpt/meta/css/cssom-view/scrollWidthHeight-negative-margin-002.html.ini

```diff
@@ -121,97 +121,97 @@
     expected: FAIL
 
   [scrollWidth with negative margins: display: flex; overflow: visible; direction: ltr; flex-direction: row;]
-    expected: FAIL
+    expected: [FAIL, PASS]
```

Comment:
Wait, why are the results are unreliable?

Reply:
The test uses `cssText` API, and because some CSS property is not enabled, the name contains some duplicates. I have added `tests/wpt/tests/css/css-overflow/scrollable-overflow-transform-unreachable-region.html` test which test quite similar case without  this dependency.